history/characters/1_azerothian.txt

```diff
@@ -9029,118 +8813,5136 @@
+	father=1275	#Gavin
+	582.12.19={ birth=yes add_trait=creature_human }
+	656.7.4={ death=yes }
+}
+1280={
+	name=Darrius
+	dynasty=11
+	dna="jaejigjkfen"
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=8 stewardship=6 intrigue=8 learning=6
+	add_trait=tough_soldier add_trait=paranoid add_trait=kind add_trait=cruel add_trait=lustful
+	add_trait=craven add_trait=arbitrary
+	father=1275	#Gavin
+	585.5.24={ birth=yes add_trait=creature_human }
+	626.2.11={ death=yes }
+}
+1281={
+	name=Ripley
+	dynasty=11
+	dna="gonedcdbmep"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=4 stewardship=6 intrigue=5 learning=5
+	add_trait=charismatic_negotiator add_trait=proud add_trait=temperate add_trait=greedy add_trait=diligent
+	father=1278	#Luke
+	602.6.11={ birth=yes add_trait=creature_human }
+	695.8.25={ death=yes }
+}
+1282={
+	name=Nico
+	dynasty=11
+	dna="giedbcbjgam"
+	culture=azerothian religion=holy_light
+	martial=8 diplomacy=5 stewardship=8 intrigue=7 learning=7
+	add_trait=flamboyant_schemer add_trait=scholar add_trait=wroth add_trait=content add_trait=slothful
+	add_trait=cruel
+	father=1278	#Luke
+	608.10.27={ birth=yes add_trait=creature_human }
+	698.7.16={ death=yes }
+}
+1283={
+	name=Luke
+	dynasty=11
+	dna="ikgdpjjjgle"
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=8 stewardship=6 intrigue=5 learning=7
+	add_trait=misguided_warrior add_trait=diligent add_trait=gregarious add_trait=greedy add_trait=temperate
+	father=1278	#Luke
+	612.6.3={ birth=yes add_trait=creature_human }
+	717.11.30={ death=yes }
+}
+1284={
+	name=Carletta
+	female=yes
+	dynasty=11
+	dna="lijdckjkkjp"
+	culture=azerothian religion=holy_light
+	father=1278	#Luke
+	619.3.17={ birth=yes add_trait=creature_human }
+	715.5.24={ death=yes }
+}
+1285={
+	name=Arcadio
+	female=yes
+	dynasty=11
+	dna="palenccdjep"
+	culture=azerothian religion=holy_light
+	father=1281	#Ripley
+	634.10.26={ birth=yes add_trait=creature_human }
+	724.5.5={ death=yes }
+}
+1286={
+	name=Tristen
+	dynasty=11
+	dna="goldlnkbmdf"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=7 stewardship=4 intrigue=4 learning=5
+	add_trait=martial_cleric add_trait=honest add_trait=wroth add_trait=lustful add_trait=diligent
+	father=1281	#Ripley
+	638.2.1={ birth=yes add_trait=creature_human }
+	728.5.28={ death=yes }
+}
+1287={
+	name=Gisela
+	female=yes
+	dynasty=11
+	dna="gojefddpmgc"
+	culture=azerothian religion=holy_light
+	father=1281	#Ripley
+	644.10.6={ birth=yes add_trait=creature_human }
+	698.9.2={ death=yes }
+}
+1288={
+	name=Joseph
+	dynasty=11
+	dna="noipdhdbapb"
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=8 stewardship=5 intrigue=6 learning=4
+	add_trait=mastermind_theologian add_trait=trusting add_trait=just add_trait=cruel add_trait=lustful
+	add_trait=brave add_trait=cynical
+	father=1281	#Ripley
+	650.6.5={ birth=yes add_trait=creature_human }
+	735.4.8={ death=yes }
+}
+
+# dynasty=12 Millstipe Nobles of Darkshire
```

Comment:
It is ok that, like, all of them are forsaken?

Reply:
Yes. Mentioned it in the changelog.